HOWTO import 'scripts' from Google Docs

lperson opened the pull request:
# Fixes #1237 

Supersedes #1248.  I started with @joemcl's draft in that PR and filled in the details he asked me to provide.

## Description

Documentation for script import from Google Docs

# Checklist:

- [x] I have manually tested my changes on desktop and mobile
- [ ] The test suite passes locally with my changes
- [ ] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [x] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
- [x] I have made any necessary changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] My PR is labeled [WIP] if it is in progress

ibrand commented:
@lperson should I wait for @joemcl to review this and maybe add more changes? Or should we merge it and then let me him edit and add more stuff? We're now merging PRs that are exclusively documentation straight to main :)

lperson commented:
@ibrand Did you read it?  If you're good with it, I think we should merge it.  @joemcl can do a pull request to enhance it.  I fixed the formatting a bit and removed the @ mentions.
